Building a Multi-class Text Classification model using H2O and Sckit-learn

charles721 opened the pull request:
### Article checklist

## Attention
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

#### Pre-submission checks
> Please ensure that you have only one open topic suggestion (issue) + in-review article (linked pull request) at a time. This will ensure that we complete the review process in a timely manner from inception to publishing

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

#### Topic suggestion this closes
> Remove the backticks and add the issue number below to link and close your Topic Suggestion (issue) when your article has been published (PR has been merged). See this [video](https://youtu.be/TKJ4RdhyB5Y?t=278) for more details.

This closes #6725

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

ahmadmardeni1 commented:
Plagiarism check done ✅

mercymeave commented on content/articles/building-a-multi-class-text-classification-model-using-h2o-and-sckit-learn/index.md:
```diff
@@ -0,0 +1,437 @@
+
+The image shows the companies with customer complaints.
+
+#### Renaming `Consumer complaint narrative` column
```
Why you please help the readers understand why you perform renaming?

mercymeave reviewed (changes requested):
Hello @charles721 

I had problems running your code on Colab, I kept getting the error, "session crashed after using all available ram"

 Would you please do something about this?

charles721 commented:
Hello @mercymeave, about this error "session crashed after using all available ram", I was running ten models that is why it was taking much  RAM. It also takes a lot of time if the process manages to run to completion (about three hours).

So I have made some adjustments and now I am running five models. I have added some new explanations in the article files to meet these new changes. 

Also, I have added a section that explains to the reader how to connect to Google's Colab GPU. The GPU is faster and it will save time running the five models. Without the GPU it took about 1.5hrs.

I have passed some new parameters. I have explained to the reader the reason for doing so. We were using a very large dataset, the new parameters will ensure we only use a portion of the dataset so that the whole process runs to completion without the session crashing.

I have rerun the whole model it took about 46 minutes to run the five models.  You can try to run the code in Google Colab now that I have made the adjustments.

Thanks.

![runned-five-models](https://user-images.githubusercontent.com/88342561/159530059-1274041b-fa8f-47f2-9515-192a6bfd2fdc.png)
